2.0/trigger_daily_flow.py

```
@@ -0,0 +1,26 @@
+import asyncio
+import logging
+from datetime import datetime, timedelta
+from prefect.client import get_client
+import config.settings as settings
+from config.log_config import config_logging
+
+async def trigger_yesterday_flow():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)